RecoTracker/LSTCore/src/alpaka/Segment.h

```diff
@@ -642,20 +642,20 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
       }
       alpaka::syncBlockThreads(acc);
 
-      // Occupancy matrix for 0.8 GeV pT Cut
+      // Occupancy matrix for 0.8 GeV pT Cut, 99.9%
       constexpr int p08_occupancy_matrix[4][4] = {
-          {572, 300, 183, 62},  // category 0
-          {191, 128, 0, 0},     // category 1
-          {0, 107, 102, 0},     // category 2
-          {0, 64, 79, 85}       // category 3
+          {497, 184, 144, 47},  // category 0
+          {516, 350, 0, 0},     // category 1
```

Comment:
191->516 or 128->350 are pretty large changes; I can't think of large changes in the LS code/selections. Do we have any plots/slides available supporting the old values?
 @YonsiG

Reply:
![Screenshot 2025-01-24 at 5 56 17 PM](https://github.com/user-attachments/assets/a51ca9eb-e956-44d2-a702-9fc4de8fbabd)
![Screenshot 2025-01-24 at 5 56 45 PM](https://github.com/user-attachments/assets/0d80ade9-651a-474d-97f3-977de725c6f0)

Nice catch. The percentiles she used seem to vary here, I will fix this.

Reply:
Hi Gavin, sorry for coming to this late. Yes, I didn't pick the same quantile value everywhere for the regions. And I chose the value seems to depend on if I see a performance loss after the cut. I'm not sure the performance change will be the same case now though..